### DISCUSSION TITLE
make a global property

### DISCUSSION THREAD

**Discussion by shahensaifullah** [nuxt/nuxt#17334] [General]
![image](https://user-images.githubusercontent.com/44544213/177539623-303f4497-77bc-4544-8ebe-5e5138164f42.png)

in ohmyfetch, i want "apiFetch" will be a global function.
 how can I do that?

**Comment by Flanker72** [comment]
You can create you own [composable](https://v3.nuxtjs.org/guide/directory-structure/composables), which will be auto-imported and available globally.
Also you can have a look at https://github.com/nuxt/framework/discussions/5370 about how to make a wrapper over the standard useFetch.

**Comment by mihail727** [comment]
maybe that will help you => https://github.com/nuxt/framework/discussions/4597
